BrowserKit/Sources/SiteImageView/ImageProcessing/BundleImageFetcher/BundleImageFetcher.swift

```diff
@@ -4,8 +4,8 @@
 
 import UIKit
 import Common
-// Ecosia: Import Core
-import Core
+// Ecosia: Import Ecosia Framework
+import Ecosia
```

Comment:
That's actually causing a compile error when checking out the branch

<img width="957" alt="Screenshot 2024-12-09 at 15 41 52" src="https://github.com/user-attachments/assets/922468e5-d0a7-4fce-bae4-1b7655b428f7">

Reply:
I guess Ecosia must be defined as dependency to BrowserKit and/or it's libraries

Reply:
It seems there are some absolute paths defined when building EcosiaFramework. I could not successfully build it when I had two XCode.app on the computer. Only after deleting XCode.app (16.1), I could compile it with XCode15.4.app. I could not pinpoint the issue, but it works now with a single xcode on the machine. Not sure if this will fly on CI though